lib/controllers/page_controller.rb

```diff
@@ -85,7 +88,7 @@ def restrict_concealed(page)
   end
 
   get '/:slug/edit' do
-    @page = Page.find(slug: slug)
+    @page = Page.with_concealed_if(starkast?).find(slug: slug)
```

Comment:
I assume this was changed by mistake, had to change it back in https://github.com/Starkast/wikimum/commit/257af591af3951368457e7586b32c064db112d7a

Error was

```
2022-04-22 10:52:14 - NoMethodError - undefined method `title' for #<Enumerator:0x00007fab9072d830>:
/app/lib/controllers/page_controller.rb:92:in `block in <class:PageController>'
```

We now have tests for this route :-)

Reply:
Correct.

> We now have tests for this route :-)

👍